app/models/event.rb

```diff
@@ -154,6 +154,15 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # locks the ability to change application statuses
+  def lock_application_status
+    update(application_status_locked: true)
+  end
+  # unlocks the ability to change application statuses
+  def unlock_application_status
```

Comment:
Concerning unlock: Alex R and I will talk with our PO about this today. We will get a new user story today which has the new status system (Nachrücker etc), so our stuff probably will be changed again. (If unlock is superfluous to our issue we can remove it / move it to tests)

Send Mail buttons are always visible btw, hiding them was not asked in the user story (and would be not good at the moment, since locking happens on the Event Applicants Overview page, not the Mail page, so you could lock the event without yet sending the Mails or only one kind)